src/public/map.js

```diff
@@ -11,6 +11,8 @@ var maxBounds; // lat long range of entire map
 var oms; // Overlapping Marker Spiderfier
 var totalCount;
 var currentCount;
+var shapes;
+var regionKeys;
```

Comment:
regionKeys is an object to keep track of objects (L.polygon) drawn from the region drop down so they can be deleted when their box is unchecked. I tried to save the object reference inside of selections.regions but JSON.parse was giving a 'TypeError: cyclic object value' so regionKeys was made to store the object reference instead of storing it in selections.regions.